sanity.cli.ts

```diff
@@ -10,4 +11,13 @@ loadEnvConfig(__dirname, dev, { info: () => null, error: console.error })
 const projectId = process.env.NEXT_PUBLIC_SANITY_PROJECT_ID
 const dataset = process.env.NEXT_PUBLIC_SANITY_DATASET
 
-export default defineCliConfig({ api: { projectId, dataset } })
+export const client = sanityClient({
```

Comment:
there's a client in `client.ts` are we using that one

Reply:
I'm seeing an issue with `client` in `components/IFramePreviewView.tsx`. 
you'd want to import the client you define here. 
But only after you use `createClient` instead of `sanityClient` as constructor.
Also `createClient` needs ot be imported from `next-sanity` package, not `@sanity/client`